code-challenges.test.js

```diff
@@ -10,58 +10,160 @@
+
+    //   ● wordCap › returns an array with all the words capitalized.
+
+    // //     ReferenceError: wordCap is not defined
+
+//   const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // Pseudo code:
+    // input: array
+    // output: returns an array with all the words capitalized.
+    // process: create a function name: wordCap
+    // use array as parameter
+    // return use .map method on array
+    // declare using value and replace value with 
+    // 0 index of value with .toUpperCase() method and also .substring() method starting on index 1 
+    // test pass.
+
+// // b) Create the function that makes the test pass.
+
+    // const wordCap = (array) => {
+    //     return array.map((value) => value[0].toUpperCase() + value.substring([1]))
+    // }
+
+// // Pass result:
+    // PASS  week-2-assessment/code-challenges.test.js
+    //   wordCap
+    //     ✓ returns an array with all the words capitalized. (4 ms)
+
+// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
-// Expected output: 1
-const vowelTester2 = "academy"
-// Expected output: 0
-const vowelTester3 = "challenges"
-// Expected output: 2
+    // describe("firstVowel", () => {
+    //     it("the index of the first vowel.", () => {
+    //       expect(firstVowel(vowelTester1)).toEqual(1),
+    //       expect(firstVowel(vowelTester2)).toEqual(0),
+    //       expect(firstVowel(vowelTester3)).toEqual(2)
+    //     })
+    //   })
+
+// Good Fail:
+    // FAIL  week-2-assessment/code-challenges.test.js
+    //   firstVowel
+    //     ✕ the index of the first vowel. (3 ms)
+
+    //   ● firstVowel › the index of the first vowel.
+
+    //     ReferenceError: firstVowel is not defined
+
+
+// const vowelTester1 = "learn"
+// // Expected output: 1
+// const vowelTester2 = "academy"
+// // Expected output: 0
+// const vowelTester3 = "challenges"
+// // Expected output: 2
+
+// // Pseudo code:
+    // input: string
+    // output: the index of the first vowel.
+    // process: create a function name: firstVowel
+    // create a new array: vowel = ["a", "e", "i", "o", "u"] in the function
+    // then use for loop decalre starting point ; stop point; 
+    // for (let i = 0; i < string.length; i++)
+    // using conditional operator if
+    // use.includes method with string[i] to do vowel check at each index
+    // then return (i) for index of first vowel in the string
+    // test to see if it pass.
```

Comment:
Pseudo code is a little confusing; I would reword it to make it more clear that you are looking at each index of the inputted string to see if that letter is included in the vowels array, then returning that index.